src/VirtoCommerce.DescriptionExportImportModule.Core/Services/IProductDescriptionSearchService.cs

```diff
@@ -0,0 +1,10 @@
+using System.Threading.Tasks;
+using VirtoCommerce.DescriptionExportImportModule.Core.Models;
+
+namespace VirtoCommerce.DescriptionExportImportModule.Core.Services
+{
+    public interface IProductDescriptionSearchService
```

Comment:
Why naming does not match the domain model's name? I mean EditorialReview or just Review.